server/netty-server/src/main/scala/sttp/tapir/server/netty/NettyConfig.scala

```diff
@@ -24,13 +25,30 @@ import sttp.tapir.server.netty.NettyConfig.EventLoopConfig
   * @param initPipeline
   *   The function to create the Netty pipeline, using the configuration instance, the pipeline created so far, and the handler which
   *   contains tapir's server processing logic.
+  *
+  * @param requestTimeout
+  *   Raises a ReadTimeoutException when no data is read from Netty within the specified period of time for a request.
+  *
+  * @param connectionTimeout
+  *   Specifies the maximum duration within which a connection between a client and a server must be established.
+  *
+  * @param socketTimeout
+  *   Refers to the duration for which a socket operation will wait before throwing an exception if no data is received or sent.
```

Comment:
so a socket timeout effectively also establishes a read timeout?

Reply:
Yes, exactly

Reply:
it would be good to mention this in the scaladocs, then :)